windows/jitsi-meet/index.html

```diff
@@ -0,0 +1,53 @@
+ <!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            body, html {
+                margin: 0;
+                height: 100%;
+                overflow: hidden;
+            }
+            body {
+                -webkit-app-region: drag
+            }
+            iframe {
+                position: relative;
+                width: 100%;
+                height: 100%;
+                border: 0 none;
+            }
+        </style>
+        <link rel="stylesheet" href="../../css/welcome.css">
+        <link rel="stylesheet" href="../../css/animate.css">
+        <script src="https://meet.jit.si/external_api.js"></script>
+        <script>
+            require("./render.js");
+        </script> 
+    </head>
+    <body>
+        <div id="background_pattern"></div>
+        <div id="background_logo"></div>
+        <div id="cs-loader">  
```

Comment:
Let's cleanup everything here. Everything is going to be replaced by the calendar view, so just show the background a logo.